Brief4/CSS/Mycss.css

```diff
@@ -298,6 +298,11 @@ label{
 .video{
   margin-top: 90px;
   margin-left: 100px;
+  display: flex;
+  justify-content: center;
+}
+.video img{
+	margin-right: 5px;
```

Comment:
what's the utility

Reply:
Display flex, with justify-content : center makes sure the container's inner content gets positioned in the centre of the container.
The margin-right:5px, to add a space between the image elements